src/translations/en.json

```diff
@@ -2,7 +2,7 @@
   "panel": {
     "energy": "Energy",
     "calendar": "Calendar",
-    "config": "Configuration",
+    "config": "Settings",
```

Comment:
🎉

Reply:
We need to reset that key from Lokalise if we change this.

Maybe is wiser to change the key as well?

Reply:
To change the key we would need to update the hass.panels data